src/h5web/vis-packs/core/containers/LineVisContainer.tsx

```diff
@@ -19,20 +22,28 @@ function LineVisContainer(props: VisContainerProps) {
   const { shape: dims } = entity;
   const [dimMapping, setDimMapping] = useDimMappingState(dims, 1);
 
+  const { evictCancelledValues } = useContext(ProviderContext);
+
   return (
     <>
       <DimensionMapper
         rawDims={dims}
         mapperState={dimMapping}
         onChange={setDimMapping}
       />
-      <Suspense fallback={<ValueLoader message="Loading entire dataset" />}>
-        <MappedLineVis
-          valueDataset={entity}
-          dims={dims}
-          dimMapping={dimMapping}
-        />
-      </Suspense>
+      <ErrorBoundary
+        resetKeys={[dimMapping]}
+        FallbackComponent={ErrorFallback}
+        onError={() => evictCancelledValues()}
+      >
```

Comment:
Just adding an `ErrorBoundary` in this container as well. The only containers without a boundary (and that rely on the `Visualizer`'s boundary) are now the `RawVisContainer` and `ScalarVisContainer`. Not sure yet how I'll deal with them in the containers' refactoring.